src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -49,6 +51,8 @@ public class FinancialTransactionCategory implements Serializable {
     @Column(name = "creation_date")
     @DateTimeFormat(pattern = "yyyy-mm-dd hh:mm:ss")
     private Instant creationDate;
+    @OneToMany(mappedBy = "financialTransactionCategory")
```

Comment:
Insert one blank line below, please. Mean there should be an empty line:
- between every field in the class,
- between every method in the class,
- import and class declaration
- between the last method declaration character `}` and the end of the class declaration character `}`

Reply:
done, thank you